vue/components/ui/molecules/lineup/lineup.vue

```diff
@@ -11,7 +11,7 @@
                 class="lineup-item-container"
                 v-bind:style="lineupItemStyle"
                 v-bind:class="`lineup-item-container-${value.value}`"
-                v-for="value in values"
+                v-for="value in fields"

```

Comment:
`field in fields`

Also, check that the HTML classes also reflect this name change.